examples/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -0,0 +1,147 @@
+"""
+===================================
+AIA to STEREO Coordinate Conversion
+===================================
+
+In this example I am going to demonstrate how you can identify a feature on
```

Comment:
...we demonstrate...

also, so don't actually identify a feature, you have just picked an area, right?